aws/table_aws_codepipeline_pipeline.go

```diff
@@ -30,13 +30,15 @@ func tableAwsCodepipelinePipeline(_ context.Context) *plugin.Table {
 				Name:        "name",
 				Description: "The name of the pipeline.",
 				Type:        proto.ColumnType_STRING,
+				Hydrate:     getCodepipelinePipeline,
+				Transform:   transform.FromField("Name", "Pipeline.Name"),
 			},
 			{
 				Name:        "arn",
 				Description: "The Amazon Resource Name (ARN) of the pipeline.",
 				Hydrate:     getCodepipelinePipeline,
 				Type:        proto.ColumnType_STRING,
-				Transform:   transform.FromField("Metadata.PipelineArn"),
+				Transform:   transform.FromField("PipelineArn", "Metadata.PipelineArn"),
```

Comment:
Why we need 2 different path, since we have added `Hydrate` in it?
```suggestion
				Transform:   transform.FromField("Metadata.PipelineArn"),
```